com.unity.render-pipelines.high-definition/Runtime/Utilities/CameraCache.cs

```diff
@@ -18,24 +18,19 @@ class CameraCache<K>: IDisposable
 
         /// <summary> Get or create a camera for the specified key </summary>
         /// <param name="key">The key to look at.</param>
-        /// <param name="frameCount">
-        /// The current frame count.
-        ///
-        /// This frame count is assigned to the returned camera to know the age of its last use.
-        /// </param>
         /// <param name="cameraType"> The type of camera to create if one does not exists.</param>
         /// <returns>
         /// The cached camera if the key was found,
         /// otherwise a new camera that was inserted in the cache during the call.
         /// </returns>
-        public Camera GetOrCreate(K key, int frameCount, CameraType cameraType = CameraType.Game)
```

Comment:
This is a public API so we need to make this one Obsolete and create a new one.

Reply:
indeed, that made test fail. I reverted the changes to the prototype because that's easier